mlflow/models/evaluation/default_evaluator.py

```diff
@@ -233,6 +233,16 @@ def _get_multiclass_classifier_metrics(
         pos_label=None,
         sample_weights=sample_weights,
     )
+    metrics.update(
+        roc_auc=sk_metrics.roc_auc_score(
+            y_true=y_true,
+            y_score=y_proba,
+            sample_weight=sample_weights,
+            average=average,
+            multi_class="ovr",
```

Comment:
Ref

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html


multi_class{‘raise’, ‘ovr’, ‘ovo’}, default=’raise’
Only used for multiclass targets. Determines the type of configuration to use. The default value raises an error, so either 'ovr' or 'ovo' must be passed explicitly.

'ovr':
Stands for One-vs-rest. Computes the AUC of each class against the rest [[3]](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#r4bb7c4558997-3) [[4]](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#r4bb7c4558997-4). This treats the multiclass case in the same way as the multilabel case. Sensitive to class imbalance even when average == 'macro', because class imbalance affects the composition of each of the ‘rest’ groupings.

'ovo':
Stands for One-vs-one. Computes the average AUC of all possible pairwise combinations of classes [[5]](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html#r4bb7c4558997-5). Insensitive to class imbalance when average == 'macro'.